app/assets/stylesheets/style.scss

```diff
@@ -867,4 +873,21 @@ label:hover {
   }
 }
 
+.notice_wrapper {
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 2000;
+  width: 100%;
+  background: rgba(137, 59, 100, 0.7);
+  .notice, .alert {
+    padding: 2rem 0;
+    text-align: center;
+    margin: 0;
+    font-size: 1.25rem;
+    font-weight: 700;
+    color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.